pages/folder/[[...folderId]]/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { Layout } from "@/src/sharing/feature-layout";
+import { FolderLayout } from "@/src/page-layout/FolderLayout";
+import { FolderToolBar } from "@/src/folder/feature-folder-tool-bar";
+import { SearchBar } from "@/src/link/ui-search-bar";
+import { ALL_LINKS_ID } from "@/src/link/data-access-link/constant";
+import { LinkForm } from "@/src/link/feature-link-form";
+import { CardList } from "@/src/link/feature-card-list";
+import { useSearchLink } from "@/src/link/util-search-link";
+import { useIntersectionObserver } from "@/src/sharing/util";
+import { useRouter } from "next/router";
+import { useMemo } from "react";
+import { getAccessTokenFromCookie } from "@/utils/getAccessToken";
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from "next";
+import { QueryClient, dehydrate, useQuery } from "@tanstack/react-query";
+import { Folder } from "@/src/folder/type";
+import { LinkRawData } from "@/src/link/type";
+import { mapLinksData } from "@/src/link/util-map";
+import fetcher from "@/src/sharing/util/axiosInstance";
+import { UserRawData } from "@/src/user/type";
+
+export async function getServerSideProps(context: GetServerSidePropsContext) {
+  const accessToken = getAccessTokenFromCookie(context);
+
+  if (!accessToken) {
+    return {
+      redirect: {
+        permanent: false,
+        destination: "/signin",
+      },
+    };
+  }
+
+  const queryClient = new QueryClient();
+
+  await queryClient.prefetchQuery({
+    queryKey: ["folders"],
+    queryFn: () =>
+      fetcher<Folder[]>({
+        method: "get",
+        url: "/folders",
+        headers: {
+          Authorization: accessToken,
+        },
+      }),
+  });
+
+  await queryClient.prefetchQuery({
+    queryKey: ["me"],
+    queryFn: () => fetcher<UserRawData[]>({ url: "/users", method: "GET" }),
```

Comment:
```suggestion
    queryKey: ["users"],
    queryFn: () => fetcher<UserRawData[]>({ url: "/users", method: "GET" }),
```

`queryKey`가 생각나시지 않으실 때는 url의 `path`를 참고하셔서 RESTful로 생각하시면 쉽게 도출됩니다 ! =)